ads/google/a4a/traffic-experiments.js

```diff
@@ -165,9 +173,24 @@ function maybeSetExperimentFromUrl(win, element, experimentName,
     '0': null, // TODO Ensure does not generate exp id
     '1': controlBranchId,
     '2': treatmentBranchId,
+    '3': hasLaunched(win, element) ? treatmentBranchId : controlBranchId, // DF
   };
   if (argMapping.hasOwnProperty(arg)) {
     forceExperimentBranch(win, experimentName, argMapping[arg]);
+    if (arg == '3') {
+      switch (element.getAttribute('type')) {
+        case 'adsense':
+          toggleExperiment(win, ADSENSE_COUNT_ON_RENDER, true, true);
+          break;
+        case 'doubleclick':
+          toggleExperiment(win, DOUBLECLICK_COUNT_ON_RENDER, true, true);
+          break;
+        default:
+          dev().warn('A4A-CONFIG', 'Unknown a4a ad type parameter: ',
```

Comment:
This will occur if there are non-adsense/doubleclick slots on the page.  Make sure it does not appear in reported logs or console